src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$sizeOfPoint: 34px;
+$sizeOfWatch: 550px;
+$borderOfPoint: 4px;
+$halfBorderRadius: 50%;
+$centringElements: $sizeOfWatch / 2;
+$centringOfPoint: $centringElements - $borderOfPoint - $sizeOfPoint / 2;
+$mainBlueColor: #0e3795;
+$defaultBackColor: #e9ecf3;
```

Comment:
don't create variable for the value that's used just once

Reply:
now i have it two times, so i didnt remove it